src/Helpers/getDefinedFilters.ts

```diff
@@ -0,0 +1,13 @@
+// tslint:disable-next-line
+import '../Extensions/Array'
+
+interface Query {
+  readonly publicKey?: string
+}
+
+export const getDefinedFilters = (worksFilters: Query = {}): Query => {
+  const definedFilters = Object.entries(worksFilters)
+    .filter(([key, value]) => value !== undefined)
+    .toObject()
```

Comment:
That makes sense. My thought was that it would eventually be used to define the filters as well. Something like: 

```ts
const filters: ReadonlyArray<string> = ['publicKey']

export const getDefinedFilters = (worksFilters: Query = {}): Query => Object.entries(worksFilters)
    .filter(([key, value]) => value !== undefined && filters.includes(key))
    .toObject()
```

Reply:
Updated this in #262. Let me know if you think it is OK.